### DISCUSSION TITLE
Why is respectImportant not documented in v3? Will it be dropped?

### DISCUSSION THREAD

**Discussion by unlocomqx** [tailwindlabs/tailwindcss#12687] [General]
`respectImportant` is documented in v1 & v2 but not in v3, although still works.
I want to use it for my plugin but I want to be sure it wasn't deprecated

Thank you

**Comment by wongjn** [comment]
For some things, if it is not documented, it may be considered deprecated.

**Comment by unlocomqx** [comment]
Btw I checked the code to be sure it's not deprecated

https://github.com/tailwindlabs/tailwindcss/blob/970f2ca704dda95cf328addfe67b81d6679c8755/types/config.d.ts#L293
